### ISSUE TITLE
How can I make the descriptions searchable?

### ISSUE THREAD

**Issue by CuteWaterBeary** [alexeiskachykhin/obsidian-image-metadata-plugin#4]
I thought this plugin was to allow images to be searchable by entering a description but neither the default Obsidian search nor Omnisearch is able to find images which have matching keywords in the description. Any suggestions or clues on how to get it to play nicely with search?

**Comment by alexeiskachykhin** [comment]
Hey @CuteWaterBeary the main goal of the plugin is to let you view and edit image metadata right in Obsidian. The metadata is stored in an image file itself unlike other approaches where metadata requires a separate note or a file. 

Default obsidian search is not capable of reading image metadata, so you definitely need plugins like Omnisearch. My guess is that Omnisearch is looking for specific metadata tags in an image that my plugin is not setting. I need some time to research how it works to come up with a solution.

Stay tuned.

**Comment by CuteWaterBeary** [comment]
Yes, your approach is much better as it doesn't clutter the vault with unnecessary files. I'll be looking forward to the searchable version!

**Comment by CuteWaterBeary** [comment]
This plugin might be helpful in seeing how to make Omnisearch pick up the data: [obsidian-ai-image-analyzer](https://github.com/Swaggeroo/obsidian-ai-image-analyzer)

**Comment by Rapha0815** [comment]
Anyway, to make it searchable would be a very interesting feature 👍🏼
